net.poedit.Poedit.json

```diff
@@ -1,21 +1,21 @@
 {
     "app-id": "net.poedit.Poedit",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.26",
+    "runtime-version": "3.30",
     "sdk": "org.gnome.Sdk",
     "command": "poedit",
     "finish-args": [
         "--share=ipc",
         "--socket=x11",
         "--filesystem=home",
-        /* Network */ 
+        /* LibSecret*/
+        "--talk-name=org.freedesktop.secrets",
```

Comment:
Why this was added btw?

Reply:
it based on your previous work https://github.com/flathub/net.poedit.Poedit/blob/43b5947c972f06a6aaf0a41ff263199707db3a31/net.poedit.Poedit.json